trillian/ctfe/storage/postgresql/schema.sql

```diff
@@ -0,0 +1,24 @@
+-- Copyright 2024 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- PostgreSQL version of the CTFE database schema
+
+-- "IssuanceChain" table contains the hash and value pairs of the issuance chain.
+CREATE TABLE IF NOT EXISTS IssuanceChain (
+  -- Hash of the chain of intermediate certificates and root certificates.
+  IdentityHash bytea NOT NULL,
```

Comment:
nit: add a comment explaining that this should always be of a given length?

Reply:
Addressed by https://github.com/google/certificate-transparency-go/pull/1618/commits/9744e0c97becb8e6f922688c7b496a1702dd9505.

The database doesn't care about the choice of hash algorithm and hash length, but you're right that this choice needs to be made once and once only for each log.